src/main/java/CarGenerator.java

```diff
@@ -0,0 +1,12 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarGenerator {
+    public List<Car> generateCars(String[] carNames) {
+        List<Car> cars = new ArrayList<>();
+        for (String name : carNames) {
+            cars.add(new Car(name));
+        }
+        return cars;
```

Comment:
Stream을 사용해보시는 건 어떨까요? map 함수가 있어서 지금보다 훨씬 간결하게 만들어집니다!
```suggestion
        return Arrays.stream(carNames)
                .map(Car::new)
                .collect(Collectors.toList());
```

Reply:
carNames 가 List 가 아니라 배열이라서 for 문을 사용했었는데 Arrays.stream() 을 사용하는 법을 배웠습니다!!